src/crypto/algorithms/megolm.ts

```diff
@@ -1447,19 +1449,17 @@ class MegolmDecryption extends DecryptionAlgorithm {
 
         if (event.getType() == "m.forwarded_room_key") {
             exportFormat = true;
-            forwardingKeyChain = content.forwarding_curve25519_key_chain;
-            if (!Array.isArray(forwardingKeyChain)) {
-                forwardingKeyChain = [];
-            }
+            forwardingKeyChain = content.forwarding_curve25519_key_chain ?? [];
```

Comment:
Arguably this check is weaker than the old one, for incoming malformed events

Reply:
Indeed, i'll see what i can do about this